lib/db/lowlevel.go

```diff
@@ -311,6 +311,8 @@ func (db *Lowlevel) dropDeviceFolder(device, folder []byte, meta *metadataTracke
 	if err != nil {
 		return err
 	}
+	defer dbi.Release()
```

Comment:
Yeah, it could. Badger panics when you commit a transaction with an open iterator, leveldb lets it slide, hence the change here since it was discovered here.